movies-app/src/components/MoviesList/MoviesList.js

```diff
@@ -0,0 +1,12 @@
+export const MoviesList = ({ movies }) => {
+  return (
+    <div>
+      <h2>Movies categories</h2>
+      <ul>
+        {movies.map((movie, index) => {
```

Comment:
Consider extracting any JS logic from JSX into separate functions, placed either insider of the component body or outside (depends on scoping rules that apply to these functions).

Reply:
Ideally, in case of a arrays, you'd have:
1. `const renderItem = (item) => ... som JSX ....;`
2. `const renderList = (list) => list.map(renderItem);`

Reply:
fixed it